.github/workflows/docker.yml

```diff
@@ -0,0 +1,69 @@
+---
+
+name: Docker
+
+on:
+  workflow_run:
+    workflows: ["Daily", "Release"]
+    types: [completed]
+
+jobs:
+  publish_docker:
+    name: Publish Docker
+    runs-on: ubuntu-latest
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+
+    strategy:
+      matrix:
+        runtime:
+          - name: daily-x86_64
+            packages: io.elementary.Platform/x86_64/daily io.elementary.Sdk/x86_64/daily
+            tags: ghcr.io/${{ github.repository }}/runtime:daily,ghcr.io/${{ github.repository }}/runtime:daily-x86_64
+          - name: daily-aarch64
+            packages: io.elementary.Platform/aarch64/daily io.elementary.Sdk/aarch64/daily
+            tags: ghcr.io/${{ github.repository }}/runtime:daily-aarch64
```

Comment:
NGTL, it seems weird to remove the publish part of the daily and stable workflows and add them here. The `workflow_run` part is pretty cool though, and this does mean that the stable image will be more up to date (also built on cron.)

Reply:
Yeah, I just wanted to be able to see how all of the docker containers were published in one workflow. But I can see the argument for either.

Reply:
Yeah. Not breaking. Just thought it a bit strange compared to what I've seen elsewhere.